Dockerfile

```diff
@@ -19,9 +19,6 @@
 # and PhantomJS
 FROM maven:3.6.3-jdk-8
 
-# Install the non-headless JRE as some tests requires them
```

Comment:
your branch is a couple of commits behind master, how will this interact with 667083e648d52c0aba7f4da600ff17e5510d9b1b by @tbouron?